### PR TITLE
Add augur (Nextstrain)

### PR DESCRIPTION
**Pull request by dfornika**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dfornika** [comment]
```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - snakemake
  - treetime
```
...probably because `treetime` depends on python2 (?)

https://github.com/bioconda/bioconda-recipes/blob/5808600756fa7a10351fdb475c2091fd1b9f97a7/recipes/treetime/meta.yaml#L22

This pull request allows `treetime` to build against python3:

https://github.com/bioconda/bioconda-recipes/pull/13135

File: recipes/augur/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  name: augur
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/n/nextstrain-augur/nextstrain-augur-{{ version }}.tar.gz
+  sha256: c112e6480456184b910b37f5c5679917eb56cc8cc80283c281725f8249c65c73
+
+build:
+  number: 0
+  noarch: python
```
Comment (bgruening) [inline recipes/augur/meta.yaml]: You can use:

script: "{{ PYTHON }} -m pip install . --no-deps -vv"

File: recipes/augur/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+requirements:
+  host:
+    - python >=3.4
+    - six
```
Comment (bgruening) [inline recipes/augur/meta.yaml]: setuptools and pathlib can be removed I hope
pip should be added

**Comment by dfornika** [comment]
@bgruening Thanks for your suggestions. Ready to merge now?

**Pull request merged**